src/component/Flatlist.js

```diff
@@ -1,27 +1,23 @@
 import React from 'react';
-import { Text , FlatList } from 'react-native';
+import {Text, FlatList} from 'react-native';
 
 const PFlatList = props => {
-    const {
-        style,
-        data,
-        renderItem,
-        horizontal,
-    } = props;
-    
-    return (
-        <FlatList 
-            keyExtractor={item => item.id}
-            renderItem={renderItem}
-            data={data}
-            {...horizontal === true ? horizontal : null}
-            horizontal
-            contentContainerStyle={{
-                flexDirection: 'column',
-                flexWrap: 'wrap'
-            }}
-        />
-    );
+  const {style, data, renderItem, horizontal} = props;
+
+  return (
+    <FlatList
+      keyExtractor={item => item.id}
+      renderItem={renderItem}
+      data={data}
+      numColumns={3} // set number of columns
```

Comment:
number of columns must be responsive number for example when i create 3 items with long text i have design bug